### DISCUSSION TITLE
Installation troubleshooting - Mac intel

### DISCUSSION THREAD

**Discussion by zhifengsun** [aclap-dev/video-downloadhelper#1057] [Help and Q&A]
### Your question

I'm having trouble installing the coapp. Every time I tried the system prompts an installation failure. What else can I try?

<img width="621" alt="Screenshot 2024-07-25 at 12 05 42 PM" src="https://github.com/user-attachments/assets/cf8edb01-9c91-42ae-bcdc-ad9f590f3d79">

### Addon & CoApp version and details

version: 9.0.2.12
target: google
channel: stable
lang: en-US
coapp: {"error":"Specified native messaging host not found.","found":false}
license: {"unset":true}
platform: x86-64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36
{
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 3
}

**Comment by paulrouget** [comment]
What's your version of MacOS?
